### DISCUSSION TITLE
0. 序言

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#152] [Comments]
# 0. 序言

序言 本章内容 算法与数据结构 算法复杂度 LeetCode 入门与攻略 LeetCode 题解（字典序排序，850+ 道题解） LeetCode 题解（按分类排序，推荐刷题列表 ★★★） LeetCode 面试最常考 100 题（按分类排序） LeetCode 面试最常考 200 题（按分类排序）

[https://algo.itcharge.cn/00_preface/](https://algo.itcharge.cn/00_preface/)

**Comment by darstib** [comment]
新手友好型指南~

**Comment by Head-In-The-Cloudy** [comment]
hi

**Comment by youi040804** [comment]
我嘞个去，看我发现了什么宝藏。立马收藏
